.github/workflows/check-release.yml

```diff
@@ -0,0 +1,32 @@
+name: Check Release
+on:
+  push:
+    branches: [ master, main ]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  check_release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Base Setup
+        uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
+
+      - name: Install Dependencies
+        run: |
+          pip install -e .
```

Comment:
Normally we shouldn't need this as it might add side-effects, and this step does not exist during the actual release process.

Reply:
Ok. Maybe we should remove it from the extension-template repo as well?

Reply:
Ah if it's there then yes :+1:

Reply:
Yes, it is in the template. I have not changed anything from the template except replacing Jinja2 vars.

Reply:
Just opened https://github.com/jupyterlab/extension-template/pull/38